examples/pennylane/2_Graph_optimization_with_QAOA/2_Graph_optimization_with_QAOA.ipynb

```diff
@@ -22,17 +22,20 @@
    "cell_type": "markdown",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    n_nodes = 4
</pre><p>Why does the problem need to be shrunk here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/amazon-braket/amazon-braket-examples/pull/372/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>The optimization run for this problem is done on the LocalSimulator, so shrinking the problem cuts the local runtime roughly in half. Do you feel the demonstration is less valuable/understandable at the smaller size?</p>

Reply:
Ah, surprised that it makes a big enough difference for the local simulator here between 4 and 6 qubits. Are you sure the time savings isn't coming from the fewer number of iterations?

Reply:
The number of iterations for this example is still 10 - it hasn't been changed here. The problem size is reduced by shrinking the number of nodes (from 6 to 4) and layers (from 4 to 2).